packages/spacecat-shared-rum-api-client/test/common/rum-bundler-client.test.js

```diff
@@ -102,6 +162,7 @@ describe('Rum bundler client', () => {
     const containsCorrectData = result.every((item) => item.events.length === 1 && item.events[0].checkpoint === 'good');
 
     expect(result.length).to.equal(dates.length);
+    // eslint-disable-next-line no-unused-expressions
```

Comment:
somehow `eslint` is blocking me from committing, so I've to add these, even though this is not related to my change.